package-lock.json

```diff
@@ -0,0 +1,8142 @@
+{
```

Comment:
Our current belief is it makes (obviously) sense to commit end-project lockfiles (e.g. `examples/package-lock.json`), but not library projects lockfiles. I think this file should be ignored. See e.g. https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514

Mainly, it would "hide" end user failures when a new cross-dep is updated but the lock file is not -> tests are green, unlocked (or freshly installed and locked) user app builds broken.